render.py

```diff
@@ -17,7 +18,7 @@
 import sumolib
 
 
-def display_network(net: sumolib.net.Net, stats: ET.ElementTree, max_size: int):
+def display_network(net: sumolib.net.Net, stats: ET.ElementTree, centre, args, max_size: int):
```

Comment:
You are not using `centre` anymore, and `args` could be replaced by `net_name` as that is the only thing you are using it for anyway.

Remember to update the doc string too and add type hints.

Reply:
Yep, it was a mistake removing centre, it's back in 0e8eefd

Reply:
Why did you make it legend method?

Reply:
Not sure - brainfart! Fixed in 188a4c4

Also draws the centre correctly, as it should happen before the image flipping.

Reply:
Docstrings and signature fixed in d461632